File: Programowanie Aplikacji Mobilnych/klasa 4/Java + Xml/zadanieguziki/app/src/main/java/com/example/zadanie_guziki/MainActivity.java
package com.example.zadanie_guziki;

import android.graphics.Color;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });



        Button guzik1 = findViewById(R.id.btn1);
        guzik1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
               View layout = findViewById(R.id.main);
               layout.setBackgroundColor(Color.BLUE);
            }
        });

        Button guzik2 = findViewById(R.id.btn2);
        guzik2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                guzik2.setText("klik");
            }
        });
        Button guzik3 = findViewById(R.id.btn3);
        guzik3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
                // System.exit(0);
            }
        });
    }

}

File: Programowanie Aplikacji Mobilnych/klasa 4/Kotlin + Xml/12_01_fragmenty/app/src/main/java/com/example/a12_01_fragmenty/MainActivity.kt
package com.example.a12_01_fragmenty

import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        val firstFragment = Fragment_1()
        val secondFragment = Fragment_2()

        supportFragmentManager.beginTransaction().apply {
            replace(R.id.flFragment, secondFragment)

            commit()
        }

        val btn1 = findViewById<Button>(R.id.btnF1)
        val btn2 = findViewById<Button>(R.id.btnF2)

        btn1.setOnClickListener{
            supportFragmentManager.beginTransaction().apply {
                replace(R.id.flFragment, firstFragment)
                addToBackStack(null)
                commit()
            }
        }

        btn2.setOnClickListener{
            supportFragmentManager.beginTransaction().apply {
                replace(R.id.flFragment, secondFragment)
                addToBackStack(null)
                commit()
            }
        }


    }
}